pynestml/frontend/pynestml_frontend.py

```diff
@@ -117,12 +119,32 @@ def install_nest(target_path: str, nest_path: str) -> None:
     nest_path : str
```

Comment:
Could you remove the type annotations from the docstring here? These are redundant with Python type annotations now.

Reply:
This will be done in another PR, because all the functions and methods have the redundant annotations.